src/entity/device.rs

```diff
@@ -246,6 +249,22 @@ impl FileDevice {
     }
 }
 
+/// Inspects a physical device.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct PhysicalDeviceInspection {
```

Comment:
I favor more specific name than "inspection".
I'm worried that users cannot guess the functionality by the name, because "inspection" is a too generic term.
(Besides, this RPC returns opaque data...)

Reply:
This PRC is intentionally generic but I'll rename it to this use case if you say "too generic".